release-engineering/role-handbooks/release-image-promotion.md

```diff
@@ -32,12 +32,24 @@ The images are promoted by using the `krel promote-images` command. This command
 
 Depending on the release you're cutting, you need to run the following command:
 
-- If you're cutting Alpha, Beta, or RC release, use the following `krel promote-images` command to promote images for the release you're cutting:
+- If you're cutting Alpha or Beta release, use the following `krel promote-images` command to promote images for the release you're cutting:
```

Comment:
nitpick: do you think it would be worth generalizing this section and just saying "this is the command syntax", then following with notes about the special cases of first rc and stable versions requiring two tag args?

Reply:
@hasheddan The command syntax is described in detail above. Let me see can I come up with something.

Reply:
Right, I am just thinking we are pretty verbose here, which maybe is good? But maybe we could make it a little more succinct. Happy to go ahead and move forward if you prefer the more verbose approach :)

Reply:
@hasheddan I've updated the PR, PTAL. This should be more concise and easier to read, but also (hopefully) verbose enough.